src/assets/toolkit/styles/utils/position.css

```diff
@@ -1,5 +1,13 @@
 @import "suitcss-utils-position";
 
+.u-posAbsoluteMiddle {
```

Comment:
Yes, it just vertically aligns the element, and doesn't do anything with left-to-right alignment. I having an issue in which I wanted an element to span the full width, and it was being compacted by the left positioning. I might be able to fix this by altering properties on the element though. I was planning to go back through these classes with fine-toothed comb, because some of them work might be WIP that aren't relevant anymore.